docker-compose.yml

```diff
@@ -307,6 +309,27 @@ services:
     networks:
       apps:
 
+  sms-provider-stub:
+    image: sms-provider-stub
+    container_name: sms-provider-stub
+    profiles:
+      - sms-provider-stub
+    build:
+      context: ../notifications-sms-provider-stub
+      dockerfile: docker/Dockerfile
+    ports:
+      - "6300:6300"
+    stdin_open: true
+    tty: true
+    environment:
+      - FIRETEXT_CALLBACK_URL=http://notify-api.localhost:6011/notifications/sms/firetext
+      - MMG_CALLBACK_URL=http://notify-api.localhost:6011/notifications/sms/mmg
+      - USE_ECS_APPS=true
```

Comment:
this looks unusual, how come we need this? 🤔

Reply:
On the USE_ECS_APPS, that shouldn't be there and I'll rebase removing that line.

I think your comment is saying to all 3 of these lines are unusual, right? If so, could you elaborate on why passing a firetext and mmg callback url to the sms stub might be unusual? Or is it that it would be done in a different way?

I did consider a `.env` file for it but feels like overkill as people won't need to tweak it often and there are very few environment variables to pass

Reply:
Ah no, I was just saying the `USE_ECS_APPS` is unusual. The rest is good 👍

Reply:
Thanks